modules/live-schedule-impl/src/test/java/org/opencastproject/liveschedule/impl/LiveScheduleServiceImplTest.java

```diff
@@ -890,6 +890,11 @@ public boolean hasPermission(MediaPackage mp, String action) {
       return false;
     }
 
+    @Override
+    public boolean hasPermission(AccessControlList mp, String action) {
```

Comment:
Consider changing the AccessControlList variable from "mp" to "acl" to avoid confusion.